smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -658,29 +658,75 @@ def updateDevice() {
 	def data = request.JSON
 	def command = data.command
 	def arguments = data.arguments
-
+  def type = params.deviceType
 	log.debug "updateDevice, params: ${params}, request: ${data}"
 	if (!command) {
 		render status: 400, data: '{"msg": "command is required"}'
 	} else {
 		def device = allDevices.find { it.id == params.id }
-		if (device) {
-        	if (device.hasCommand("$command")) {
+    if (device) {
```

Comment:
because I'm using render status instead of httpError and I'm not sure that will stop execution of the function. The if will proceed if device and if not render status: 403, data: '{"msg": "Access denied. This command is not supported by current capability.")}' to harmony's API